### DISCUSSION TITLE
pageLoadTimeOut while downloading a xlsx file on click

### DISCUSSION THREAD

**Discussion by hafiz-usman-cowlar** [cypress-io/cypress#23784] [Questions and Help]
![image](https://user-images.githubusercontent.com/107095898/189703453-7772ec87-220a-424c-a216-4c551ba058fc.png)
When I click on downloads button, it downloads a xlsx file. It visits this [url](https://idt.dev.viaphoton.dev/data/files/viaPhoton_Panel-Labeling-HDCE-12-UA-FMEXT-RD.xlsx) to download file. After downloading cypress Error occurs. pageLoadTimeOut. 

If this issue is fixed, then plz help me to solve this issue. I searched about this issue, but not found any solution

**Comment by Priya-LaunchIT** [comment]
Check this post

https://stackoverflow.com/questions/68286691/cypress-identify-the-downloaded-file-using-regex
